az-core/src/main/java/azkaban/Constants.java

```diff
@@ -33,12 +33,16 @@
 public class Constants {
 
   // Azkaban Flow Versions
-  public static final String AZKABAN_FLOW_VERSION_2_0 = "2.0";
+  public static final String AZKABAN_FLOW_VERSION = "Azkaban-Flow-Version";
```

Comment:
this seems to be a property key name, so should it be a inside configurationkey.java?

Reply:
Yes you are right. I'll move it inside ConfigurationKeys class.